intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -2612,8 +2684,68 @@ def convert_date_utc(value: str) -> Optional[str]:
     },
 }
 
-# https://www.shadowserver.org/what-we-do/network-reporting/microsoft-sinkhole-events-report/
-# https://www.shadowserver.org/what-we-do/network-reporting/sinkhole-events-report/
+event4_honeypot_http_scan = {
```

Comment:
Thanks for your efforts on this mapping, but importunately and thanks to #2047 we already have this mapping (including tests).

Reply:
nice